app/code/Magento/MediaGalleryApi/Model/File/Command/DeleteByAssetIdInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryApi\Model\File\Command;
+
+/**
+ * Remove the media asset file from the media storage
+ * @api
+ */
+interface DeleteByAssetIdInterface
+{
+    /**
+     * Remove the file of the media asset identified by the passed id from the media storage
```

Comment:
MediaGallery should not have "file" in its domain model. It operates with "assets". So this interface also looks not as a part of MediaGallery API. This should be just implementation details.
Tomorrow we'll start saving assets in DB or in S3 objects and "files" will become irrelevant.

Reply:
Interface operating with files removed. Comment corrected